src/outlet/index.ts

```diff
@@ -184,7 +186,12 @@ export class Outlet {
 						this.dispatchAuthCallbackEvent(issuer, null, e.message)
 					}
 
-					document.location.hash = ''
+					document.location.hash = removeUrlSearchParams(this.urlParams, [
+						'attestation',
+						'requestSecret',
+						'address',
+						'wallet',
+					]).toString()
```

Comment:
Should this include the URLNS as the last parameter?

Reply:
No that parameter has a default value.